client/src/components/TinymceInlineToolbar/TinymceInlineToolbar.js

```diff
@@ -161,21 +161,41 @@ class TinymceInlineToolbar {
  * @return {Boolean}
  */
 function shouldShowToolbar(selection, isEditorFocused, tagTypes) {
-  const tags = Array.isArray(tagTypes) ? tagTypes : [tagTypes || ''];
-  return Boolean(
-    selection &&
-    tags
-      .map(tag => String(tag))
-      .map(tag => tag.toLowerCase())
-      .includes((selection.tagName || '').toLowerCase()) &&
-    isEditorFocused);
+  let tags = Array.isArray(tagTypes) ? tagTypes : [tagTypes || ''];
+
+  if (selection && isEditorFocused && Array.isArray(tagTypes) && tagTypes.length > 0) {
+    tags = tags
+        .map(tag => String(tag))
+        .map(tag => tag.toLowerCase());
+
+    const matching = tags.filter((t) => {
+      if (t.indexOf('[') > -1 && t.indexOf(']') > -1) {
+        const tTag = t.substring(0, t.indexOf('['));
+        const tAttr = t.substring(t.indexOf('[') + 1, t.indexOf(']'));
+        if (selection.tagName && (tTag === selection.tagName.toLowerCase())) {
```

Comment:
could we store `selection.tagName.toLowerCase()` to a variable?
I think `tTag` could also have `.toLowerCase()` applied to it

Could merge this `if` with the one below with an `&&`